README.md

```diff
@@ -1,6 +1,6 @@
 # Liberty Config Language Server
 
-Provides language server features for Liberty server configuration files.
+Provides language server features for Liberty server configuration files, which include `server.env`, `bootstrap.properties`, and `server.xml` files.
```

Comment:
It also includes any xml file referenced by an `include` element, and any xml file located in certain directories (confirm with UFO - but has to do with configDropins folder and the `usr/shared` folder).

Reply:
How explicit do we want to be in documenting the supported XML files?
![image](https://user-images.githubusercontent.com/689163/196283761-812f18e7-70cc-4d7c-b106-19dda7b701fa.png)

I feel like this is a lot of detail for an introduction to the language server.

Reply:
It has to be documented somewhere for the users. Each of the IDEs will link to this for their documentation. So I think it is appropriate to fully document here what is supported. Leave out the bit about the explicit schema though. We don't actually look for that.